packages/fixie/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "fixie",
+  "version": "1.0.0",
```

Comment:
Given that this is all external to the AI.JSX package I think this should probably all live in the fixie-sdk repo?

Reply:
I guess unless the eventual plan is for the the serving CLI to be in ai-jsx itself? But it's weird to see the `agent.yaml` here.

Reply:
I think it's OK to have agent.yaml outside the core ai.jsx package, it's a config file just like the next.js config files in the nextjs-demo package. Right?